src/main/java/com/brightpath/learnify/exception/notfound/ResourceType.java

```diff
@@ -0,0 +1,17 @@
+package com.brightpath.learnify.exception.notfound;
+
+import lombok.Getter;
+
+@Getter
+public enum ResourceType {
+    NOTE("note"),
+    QUIZ("quiz"),
+    NOTE_PAGE("note page"),
```

Comment:
Maybe `BOARD_PAGE` instad of `NOTE_PAGE` or even `BOARD_NOTE_PAGE` and `DOCUMENT_NOTE_PAGE`?